components/VFBMain.js

```diff
@@ -176,10 +177,7 @@ export default class VFBMain extends React.Component {
         if (idsList.length > 0) {
           this.props.vfbLoadId(idsList);
           this.fetchVariableThenRun(idsList, this.handleSceneAndTermInfoCallback);
-          this.setState({
-            UIUpdated: false,
```

Comment:
this should make it back since it's used for the ShouldComponentUpdate to minimise the not needed re-render